fluent-package/make-index-html.erb

```diff
@@ -26,15 +26,24 @@
        </a>
      </div>
      <div style="margin: 0em auto 0em 2em">
-       Index of <a href="index.html"><%= options[:site] %><%= relative_path %></a>
+       Index of <a href="index.html"><%= options[:site] %>/<%= relative_path %></a>
+     </div>
```

Comment:
```suggestion
```

Looks like this `</div>` is unintentional.

Reply:
Thanks, fixed it.